### PR TITLE
Update CN province codes for DCW 2.0.0

### LINKED ISSUE #418: DCW 数据中中国各省代码需要更新
https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/dataset/dataset/dcw/

老版本 DCW 数据中，使用两位数字作为省份编码，如：
```
CN 11 Beijing
CN 50 Chongqing
CN 31 Shanghai
CN 12 Tianjin
CN 34 Anhui
```

新版本 DCW (2.0.0 之后)数据中，使用两位字母作为省份编码，如:
```
CN.AH	Anhui
CN.BJ	Beijing
CN.CQ	Chongqing
CN.FJ	Fujian
CN.GD	Guangdong
```
这一部分需要更新。

### PR DESCRIPTION
**Pull request by seisman**
Closes #418.

### CODE REVIEW

File: source/dataset/dcw/index.rst [hunk 1]
```diff
@@ -95,46 +95,47 @@ GMT提供的DCW数据默认位于GMT安装目录下的 **share/dcw** 下，其
 
     国家代码 省代码 省名
 
-也可以使用命令 ``gmt coast -E+L`` 查看省代码。
+命令 ``gmt coast -E+L`` 可以列出 DCW 数据中的所有省代码。如果想进一步筛选出
+某个国家（如中国）的省代码，则可以使用 ``gmt coast -E+L | grep CN`` 命令。
```
Comment (core-man) [inline source/dataset/dcw/index.rst:99]:
```suggestion
使用 ``gmt coast -E+L`` 命令可以列出 DCW 数据中的所有省代码。如果想进一步筛选出
某个国家（以中国 **CN** 为例）的省代码，则可以使用 ``gmt coast -E+L | grep CN`` 命令。
```

File: source/dataset/dcw/index.rst [hunk 2]
```diff
@@ -95,46 +95,47 @@ GMT提供的DCW数据默认位于GMT安装目录下的 **share/dcw** 下，其
...
-    CN 33 Zhejiang
-    CN 45 Guangxi
-    CN 15 Nei Mongol
-    CN 64 Ningxia
-    CN 65 Xinjiang
-    CN 54 Xizang
-    CN 91 Xianggang (Hong Kong)
-    CN 92 Aomen (Macao)
+5 个自治区，4 个直辖市，以及香港，澳门 2 个特别行政区。每个省用两个字符表示
+（如 AH 表示安徽）::
```
Comment (core-man) [inline source/dataset/dcw/index.rst:103]:
```suggestion
5 个自治区、4 个直辖市以及 2 个特别行政区（香港、澳门）。
每个省用两个字符表示（如 AH 表示安徽）::
```

**Comment by core-man** [comment]
I might find an installation issue. I use `conda` to install gmt, but the `dcw` is still the old version, which is in my `~/.anaconda3/envs/gmt/share/dcw-gmt` folder.

If my understanding is right, [跨平台安装 GMT](https://docs.gmt-china.org/latest/install/cross-platform/) might need to be updated. Otherwise, the users might still use old dcw data.

**Comment by seisman** [comment]
> I might find an installation issue. I use `conda` to install gmt6.2, but the `dcw` is still the old version, which is in my `~/.anaconda3/envs/gmt/share/dcw-gmt` folder.

Need to update https://github.com/conda-forge/dcw-gmt-feedstock.

**Review by core-man — APPROVED** [review]

**Pull request merged**